FlightGroupLibraryForm.cs

```diff
@@ -0,0 +1,877 @@
+					}
+					break;
+				case Settings.Platform.XvT:
+				case Settings.Platform.BoP:
+					Platform.Xvt.FlightGroup xvt = (Platform.Xvt.FlightGroup)fg;
+					scanMothershipProblems(xvt, scrubProblems, errors);
+					for (int i = 0; i < xvt.ArrDepTriggers.Length; i++)
+					{
+						Platform.Xvt.Mission.Trigger trig = xvt.ArrDepTriggers[i];
+						if (trig.Condition != 0 && trig.Condition != 10 && (trig.VariableType == 1 || trig.VariableType == 0xF))
+						{
+							logProblem("ArrDep trigger #" + (i + 1) + " references FG:" + trig.Variable, errors);
+							if (scrubProblems) { trig.Condition = 0; trig.Variable = 0; trig.VariableType = 0; }
+						}
+					}
+					for (int i = 0; i < xvt.Orders.Length; i++)
+					{
+						Platform.Xvt.FlightGroup.Order order = xvt.Orders[i];
+						if (order.Target1Type == 1 || order.Target1Type == 0xF) { logProblem("Order #" + (i + 1) + " Target1 references FG:" + order.Target1, errors); if (scrubProblems) { order.Target1 = 0; order.Target1Type = 0; } }
+						if (order.Target2Type == 1 || order.Target2Type == 0xF) { logProblem("Order #" + (i + 1) + " Target2 references FG:" + order.Target2, errors); if (scrubProblems) { order.Target2 = 0; order.Target2Type = 0; } }
+						if (order.Target3Type == 1 || order.Target3Type == 0xF) { logProblem("Order #" + (i + 1) + " Target3 references FG:" + order.Target3, errors); if (scrubProblems) { order.Target3 = 0; order.Target3Type = 0; } }
+						if (order.Target4Type == 1 || order.Target4Type == 0xF) { logProblem("Order #" + (i + 1) + " Target4 references FG:" + order.Target4, errors); if (scrubProblems) { order.Target4 = 0; order.Target4Type = 0; } }
+					}
+					for (int i = 0; i < xvt.SkipToOrder4Trigger.Length; i++)
+					{
+						Platform.Xvt.Mission.Trigger trig = xvt.SkipToOrder4Trigger[i];
+						if (trig.Condition != 0 && trig.Condition != 10 && (trig.VariableType == 1 || trig.VariableType == 0xF))
+						{
+							logProblem("Skip trigger #" + (i + 1) + " references FG:" + trig.Variable, errors);
+							if (scrubProblems) { trig.Condition = 0; trig.Variable = 0; trig.VariableType = 0; }
+						}
+					}
+					break;
+				case Settings.Platform.XWA:
+					Platform.Xwa.FlightGroup xwa = (Platform.Xwa.FlightGroup)fg;
+					scanMothershipProblems(xwa, scrubProblems, errors);
+					for (int i = 0; i < xwa.ArrDepTriggers.Length; i++)
+					{
+						Platform.Xwa.Mission.Trigger trig = xwa.ArrDepTriggers[i];
+						if (trig.Condition != 0 && trig.Condition != 10 && (trig.VariableType == 1 || trig.VariableType == 0xF))
+						{
+							logProblem("ArrDep trigger #" + (i + 1) + " references FG:" + trig.Variable, errors);
+							if (scrubProblems) { trig.Condition = 0; trig.Variable = 0; trig.VariableType = 0; }
+						}
+						if (trig.Parameter1 > 4)
+						{
+							logProblem("ArrDep trigger #" + (i + 1) + " parameter references FG:" + trig.Parameter1, errors);
```

Comment:
Shouldn't the message be (Parameter1 - 4) to get the actual FG number?

Reply:
Just checked it out.  Looks like it needs to be (trig.Parameter1 >= 5) and (trig.Parameter1 - 5)

Reply:
Fixed on my end after the merge